### PR TITLE
pr change

### PR DESCRIPTION
**Pull request by traceur91**

### CODE REVIEW

**Comment by webnovationNET** [comment]
1

**Comment by traceur91** [comment]
2

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- The content below is only a placeholder and can be replaced. -->
 <div style="text-align:center">
-  <h1>Welcome to test 123 {{ title }}!</h1>
+  <h1>Welcome to test PR change {{ title }}!</h1>
```
Comment (webnovationNET) [inline src/app/app.component.html]: why?

Comment (traceur91) [reply src/app/app.component.html]: you need to change  this

Comment (traceur91) [inline src/app/app.component.html]: needs to be changed

Comment (traceur91) [reply src/app/app.component.html]: 123

Comment (webnovationNET) [reply src/app/app.component.html]: why

Comment (webnovationNET) [reply src/app/app.component.html]: i dont think so

Comment (webnovationNET) [reply src/app/app.component.html]: 1111

Comment (traceur91) [reply src/app/app.component.html]: yes

Comment (traceur91) [inline src/app/app.component.html]: change it to pr change 2

Comment (webnovationNET) [reply src/app/app.component.html]: why

**Pull request merged**